tensor/examples/datafs-sim/sim.go

```diff
@@ -26,9 +26,9 @@ type Sim struct {
 
 // ConfigAll configures the sim
 func (ss *Sim) ConfigAll() {
-	ss.Root = errors.Log1(datafs.NewDir("Root"))
-	ss.Config = errors.Log1(ss.Root.Mkdir("Config"))
-	errors.Log1(datafs.New[int](ss.Config, "NRun", "NEpoch", "NTrial"))
+	ss.Root, _ = datafs.NewDir("Root")
+	ss.Config, _ = ss.Root.Mkdir("Config")
```

Comment:
Why are you ignoring these errors?

Reply:
this is just an end-user test., and doesn't have any errors..